config/locales/en.yml

```diff
@@ -1131,9 +1131,7 @@ en:
         authentication_provider:
           attributes:
             site:
-              invalid: "Please, remove any white space."
-            realm:
-              invalid: "Please, remove any white space."
+              contains_whitespace: "can't contain whitespaces"
```

Comment:
I thought that the error code `invalid` is too generic for this specific error message (it can be invalid in a different way, and the message would be confusing), so I renamed it.